Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -557,6 +576,71 @@ - (void)sectionEditorFinishedEditing:(SectionEditorViewController*)sectionEditor
     [self fetchArticle];
 }
 
+
+
+
+
+
+
+
+
+
+
+
+#pragma mark - UIViewControllerPreviewingDelegate
+
+- (nullable UIViewController*)previewingContext:(id<UIViewControllerPreviewing>)previewingContext
+                      viewControllerForLocation:(CGPoint)location {
+
+//return
+//[[WMFArticleContainerViewController alloc] initWithArticleTitle:[[MWKTitle alloc] initWithURL:[NSURL URLWithString:@"http://en.wikipedia.org/wiki/Cat"]]
+//                                                      dataStore:self.dataStore
+//                                                discoveryMethod:self.discoveryMethod];
+
+
+    
+    JSValue *element = [[self.webViewController.webView wmf_javascriptContext][@"getElementFromPoint"] callWithArguments:@[@(location.x), @(location.y)]];
```

Comment:
you're probably planning on doing this already, but everything from here down to line 626 should be refactored into WebVC, which could expose a higher-level method like `- (nullable NSDictionary*)webViewElementAtPoint:(CGPoint)`